src/99-appendices/03-hed.md

```diff
@@ -45,35 +45,73 @@ Example:
 ```Text
 onset	duration	mycodes
 1.1	n/a	Fixation  
-1.3	n/a	Button
-1.8	n/a	Target
+1.3	n/a	Button  
+1.8	n/a	Target  
```

Comment:
Please remove trailing whitespace (or do not insert new trailing whitespace)

```suggestion
1.3	n/a	Button
1.8	n/a	Target
```

Reply:
Done sorry....